homeassistant/components/rainforest_eagle/sensor.py

```diff
@@ -0,0 +1,135 @@
+"""Support for the Rainforest Eagle-200 energy monitor."""
+from datetime import timedelta
+import logging
+
+from requests.exceptions import (
+    ConnectionError as ConnectError, HTTPError, Timeout)
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_IP_ADDRESS, CONF_SCAN_INTERVAL, ENERGY_KILO_WATT_HOUR)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+CONF_CLOUD_ID = 'cloud_id'
+CONF_INSTALL_CODE = 'install_code'
+POWER_KILO_WATT = 'kW'
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=30)
+
+SENSORS = {
+    "instantanous_demand": (
+        "Eagle-200 Meter Power Demand", POWER_KILO_WATT),
```

Comment:
This sensor type should have `device_class` entity property `'power'`.

See https://developers.home-assistant.io/docs/en/entity_sensor.html#available-device-classes.

Reply:
Returned `device_class` of `power` for the demand power sensor